frontend/package.json

```diff
@@ -30,9 +30,10 @@
     "ember-auto-import": "^1.5.3",
     "ember-autofocus-modifier": "^1.0.1",
     "ember-bootstrap": "^3.1.4",
+    "ember-bootstrap-power-select": "^2.0.0",
     "ember-changeset": "^3.3.8",
     "ember-changeset-validations": "^3.3.2",
-    "ember-cli": "~3.16.2",
+    "ember-cli": "~3.19.0",
```

Comment:
Huh? Why is this here? Also, there's a bug of powerselect in 3.19. I'd rather we stay on 3.16 since it's also LTS